Dockerfile

```diff
@@ -17,15 +17,23 @@ RUN apk --no-cache add \
     texmf-dist-latexextra \
     texmf-dist-pictures \
     texmf-dist-science \
-    wget
+    wget \
+    curl \
+    git
 
 
 RUN luaotfload-tool --update
 RUN apk --no-cache add py3-pandas
-RUN apk --no-cache add pipx
-RUN pipx install cgt-calc
-RUN pipx ensurepath
 
-WORKDIR /data
+WORKDIR /build
+
+RUN curl -sSL https://install.python-poetry.org | python3 -
```

Comment:
What do you think about installing `poetry` using `apk` instead?

Reply:
I prefer apk, it's more clear and easier to maintain.